giskard/visualization/templates/scan_report/markdown/huggingface.md

```diff
@@ -33,7 +34,6 @@ else True)|replace("\\n", "<br>")|safe }}
 {% endfor %}
 
 </details>
-<!-- line breaker -->
```

Comment:
Can we keep this? So that in cicd we can still fragment before the release

Reply:
We are not using this to fragment right now.